spec_releasenotes/releasenotes/config.yaml

```diff
@@ -0,0 +1,6 @@
+---
+encoding: utf8
+default_branch: main
+collapse_pre_releases: true
+branch_name_prefix: release/
```

Comment:
Should the release branch prefixes be different for the spec and the Python package?

Reply:
Yes, I'm still working on how to get reno to correctly find the right tags/branches for each artifact. We may have to have separate `spec_release/*` and `ast_releast/*` branch names.

Reply:
Ok, I think I've made reno find the right things. Reno keys on the tags to version each note, it ignores branches to determine which notes to apply. However, it uses branches to determine where to stop in the history for tags. So for the `stable/3.0` branch it will search for all tags that are newer than that branch and organize the included notes in the history back to `stable/3.0`.

So the question is, do we want to change our release process to use different stable branches for the AST and spec releases? I would advocate for yes, we should have something like`stable/spec/3.x` and `stable/ast/x.x' branches that we maintain, then we can update the `branch_name_prefix` and corresponding `_re` to match on them differently.

Reply:
TSC discussion today resolved to maintain a single stable branch for both the AST and specification.